src/packages/recompose/index.js.flow

```diff
@@ -112,15 +112,7 @@ declare export function withStateHandlers<
 
 declare export function withHandlers<
   Enhanced,
-  Handlers:
-    | ((
-        props: Enhanced
-      ) => {
-        [key: string]: (props: Enhanced) => Function,
-      })
```

Comment:
This is the unnecessary function branch to the `Handlers` union type removed by this PR.